Source/WebCore/page/Page.h

```diff
@@ -1440,6 +1443,8 @@ class Page : public RefCountedAndCanMakeWeakPtr<Page>, public Supplementable<Pag
     bool m_defersLoading { false };
     unsigned m_defersLoadingCallCount { 0 };
 
+    RefPtr<Node> m_lastTouchedNode;
```

Comment:
Looking at the existing member on `Page` and `EventHandler`, I think this would make more sense on `EventHandler`.

Reply:
Agreed